packages/ffe-form/less/input-group.less

```diff
@@ -1,7 +1,9 @@
 .ffe-input-group {
+    --field-line-heigth: 1.5rem;
+
     border: 0;
     position: relative;
-    padding: 0 0 calc(1lh + var(--ffe-spacing-xs));
+    padding: 0 0 calc(var(--field-line-heigth) + var(--ffe-spacing-xs));
```

Comment:
Uansett om vi bruker `1lh` eller `1.5rem` så reserveres det plass til en `.ffe-field-message` med én linje tekst. `1lh` gir ikke noe mer funksjonalitet utover `1.5rem`. Dersom det er mulig å styre reservert plass med en css custom prop (som gjort her) så kan teamene bruke den til å reservere plass til større eller mindre elementer.

Reply:
Ikke mer funksjionalitet utøver att att man ikke trenger ha ha noe forhold til den. Men att legge den in som themes som jag opdatert pr med nå kanskje er en mellomvei. 
Målet var att slippa en verdi i rem her også en annen verdi i rem så må vare i sync.